tox.ini

```diff
@@ -75,3 +69,16 @@ commands_pre =
 commands =
     isort -rc .
     black .
+
+[testenv:spark]
+description = run the test driver with {basepython}
+passenv = JAVA_HOME
+commands_pre = python -m spacy download en_core_web_sm
```

Comment:
You can leave `commands_pre` out on these two. It will default to the `commands_pre` of `testenv`